app/uk/gov/hmrc/apiplatform/modules/gkauth/controllers/actions/AnyGatekeeperRoleAuthorisationAction.scala

```diff
@@ -37,7 +38,10 @@ trait AnyGatekeeperRoleAuthorisationAction {
     Action.async { implicit request =>
       implicit val hc = HeaderCarrierConverter.fromRequest(request)
       ldapGatekeeperRoleAuthorisationService.ensureHasGatekeeperRole()
-        .recoverWith { case NonFatal(_) => ldapGatekeeperRoleAuthorisationService.UNAUTHORIZED_RESPONSE }
+        .recoverWith { case NonFatal(_) =>
+          logger.warn("LDAP Authenticate errored trying to find user, trying stride")
```

Comment:
I don't understand the "trying stride" part of the log.

Reply:
I asked for this from Toby because this is not an error but rather a possible failure case that means we must then try stride next.